README.rst

```diff
@@ -63,6 +63,10 @@ Configuring the Host Cloud
    happen with the baremetal instances booted from an empty image it speeds
    things up a bit.
 
+#. ``/etc/nova/nova.conf`` needs ``allow_duplicate_networks=true`` in the
```

Comment:
I think it was deprecated in Kilo and is false by default, possibly it is ignored in Liberty? So it is needed for OSPD7 host clouds